vdm-io/Joomla-Component-Builder
# Improvments to the export and import of JCB packges

Pull Request for Issue gh-204 .

### Summary of Changes
- Added auto versioning to all JavaScript and CSS files being added, to insure it reloads when there is an update.
- Improvements to the import & export of JCB packages
- Added a few snippets back into the base install package of JCB 

### Testing Instructions
Export a component and import into another Joomla/JCB website and then compile to see if all works as expected.

### Expected result
Successful compilation of imported component as if build on the JCB instance.

### Actual result
There were some discrepancies that caused some fields and values to diverge and become incorrect

### Other changes
- Update the url and email address
- Added a code of conduct

@alexsykes please move the discussion here, and help test these changes if you can.
